docs/source/faq.rst

```diff
@@ -256,4 +256,10 @@ Regardless of whether the ``colocate`` option is on or off for a given monitor,
 - ``data_at_coords = sim_data[monitor_name].colocate(x=x_points, y=y_points, z=z_points)`` to colocate all fields to a custom set of coordinates. Any or all of ``x``, ``y``, and ``z`` can be supplied; if some are not, the original data coordinates are kept along that dimension.
 
 
+Can you convert a lumerical script file to Tidy3D?
+-------------------------------------------------------------
```

Comment:
I think in `.rst` the underline must be the same length as the text, could you fix this? thanks